### ISSUE TITLE
Automation of tag-app-delivery repo

### ISSUE THREAD

**Issue by olivercodes** [cncf/tag-app-delivery#566]
There is discussion amongst the platforms-wg slack to automate the tag-app-delivery repo. Ideas are to use (one of) Prow, Github Actions, etc. Will update this GH Issue after we talk about it in detail in the WG Meeting.

Note: This carries over the conversation from this slack thread:

https://cloud-native.slack.com/archives/C020RHD43BP/p1707944806442839

**Comment by olivercodes** [comment]
WG Meeting Notes:

Collect automation needs across the TAG

Some ideas mentioned:
- Action for translation alert, awareness
- Assignment of issues and prs

**Comment by abangser** [comment]
I would love to add `good first issue` label to this one as I think it is concrete and focused for pick up. Do you think we should try to clarify the needed actions more in this issue or create associated issues that are clear and easy to be picked up?

**Comment by roberthstrand** [comment]
> I would love to add `good first issue` label to this one as I think it is concrete and focused for pick up. Do you think we should try to clarify the needed actions more in this issue or create associated issues that are clear and easy to be picked up?

Yes, I think opening this up as a `good first issue` without something more specific would make this harder for people to pick up.
